src/test/java/org/xstefank/FileLine.java

```diff
@@ -0,0 +1,24 @@
+package org.xstefank;
+
+public class FileLine {
+
+    private int lineNumber;
+    private String content;
+
+    public FileLine(int lineNumber, String line) {
+        this.lineNumber = lineNumber;
+        this.content = line;
+    }
+
+    public boolean exists() {
+        return lineNumber >= 1 && content != null;
+    }
+
+    public int getLineNumber() {
```

Comment:
This doesn't seems to be used anywhere. What is the point there in having FileLine class at all?